src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/security.yml

```diff
@@ -0,0 +1,85 @@
+admin_security:
+    path: /
+    methods: [GET]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\Security:index'
+        _legacy_controller: AdminSecurity
+        _legacy_link: AdminSecurity
+
+admin_security_general_save:
+    path: /general
+    methods: [POST]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\Security:processGeneralForm'
+        _legacy_controller: AdminSecurity
+        _legacy_link: AdminSecurity
+
+admin_security_session_customer:
+    path: /session/customer
```

Comment:
https://devdocs.prestashop.com/1.7/development/naming-conventions/#routes-and-paths
```
PrestaShop routes follow admin_{resources}_{action} naming structure and rules for it are:

{resources} (object) name should be in plural form (e.g. customers, products, orders).

Reply:
:exploding_head: 
I'll change it again ^^

Reply:
sorry to bother again, but paths should also be plural :sweat_smile: